README.md

```diff
@@ -1,2 +1,70 @@
-# project
-Zambezi project template
+Zambezi Project
+===============
+
+Whenever you're starting a project, there are about a million questions that need to be answered – some of them inevitable end up having to be answered for every project. This template attempts to answer the most common ones, providing a solid foundation for you to build on.
+
+Usage
+-----
+
+Do *not* fork this project, unless you intend to actually [contribute changes](CONTRIBUTING.md) which is more than welcome!
+
+To create a new project:
+
+1. Clone (or download) this project: `git clone git@github.com:zambezi/project.git your-project`
```

Comment:
Could we produce a GitHub release of this project (once PR merged) with a zip attached that contains the minimal files for getting started?  With a zip if is a much simpler method for getting started (no .git directory deleting).